cpp/bench/CMakeLists.txt

```diff
@@ -27,11 +27,10 @@ if(BUILD_CUML_BENCH)
     sg/kmeans.cu
     sg/main.cpp
     sg/rf_classifier.cu
-    sg/rf_regressor.cu
+    # sg/rf_regressor.cu
```

Comment:
Why is this one commented?

Reply:
rf_regressor is having some kind of a hang issue (not related to cuda-11) at the current HEAD itself. I haven't been able to find time to debug it. Thus, it has been commented for now. And will be enabled after it has been fixed.

Reply:
I have added a comment as well.

Reply:
I have also submitted issue #2662